modules/frontend/searchsharding.go

```diff
@@ -27,6 +29,15 @@ const (
 	defaultConcurrentRequests    = 50
 )
 
+var (
+	metricInspectedBytes = promauto.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "tempo",
+		Name:      "frontend_query_results_metrics_inspected_bytes",
```

Comment:
other metrics from this module are named "query_frontend...

e.g.  https://github.com/grafana/tempo/blob/main/modules/frontend/retry.go#L18

Reply:
I am leaning on the side of keeping the metric for now, and testing it out in dev. we can remove if it's not useful.

will rename the metric.